templates/app/aspnet-core/src/MyCompanyName.MyProjectName.DbMigrator/appsettings.json

```diff
@@ -9,28 +9,38 @@
   //</TEMPLATE-REMOVE>
   "OpenIddict": {
     "Applications": {
+      //<TEMPLATE-REMOVE IF-NOT='ui:mvc'>
       "MyProjectName_Web": {
         "ClientId": "MyProjectName_Web",
         "ClientSecret": "1q2w3e*",
         "RootUrl": "https://localhost:44302"
       },
+      //</TEMPLATE-REMOVE>
+      //<TEMPLATE-REMOVE IF-NOT='ui:blazor'>
       "MyProjectName_Blazor": {
         "ClientId": "MyProjectName_Blazor",
         "RootUrl": "https://localhost:44307"
       },
+      //</TEMPLATE-REMOVE>
+      //<TEMPLATE-REMOVE IF-NOT='ui:angular'>
       "MyProjectName_App": {
         "ClientId": "MyProjectName_App",
         "RootUrl": "http://localhost:4200"
       },
+      //</TEMPLATE-REMOVE>
+      //<TEMPLATE-REMOVE IF-NOT='ui:blazor-server&&TIERED'>
       "MyProjectName_BlazorServerTiered": {
         "ClientId": "MyProjectName_BlazorServerTiered",
         "ClientSecret": "1q2w3e*",
         "RootUrl": "https://localhost:44309"
       },
+      //</TEMPLATE-REMOVE>
+      //<TEMPLATE-REMOVE IF-NOT='HostWithIds'>
       "MyProjectName_Swagger": {
         "ClientId": "MyProjectName_Swagger",
         "RootUrl": "https://localhost:44305"
       }
+      //</TEMPLATE-REMOVE>
     }
   }
 }
```

Comment:
The json will be invalid if we remove the last one.


<img width="819" alt="image" src="https://github.com/abpframework/abp/assets/6908465/26c599ff-7364-47ee-aa1b-18610eb7b0e5">